rusk-profile/src/lib.rs

```diff
@@ -19,9 +19,10 @@ pub use theme::Theme;
 mod circuit;
 pub use circuit::Circuit;
 
-/// HEX representaion of the SHA-256 hash of the CRS uncompressed bytes.
+/// HEX representation of the SHA-256 hash of the CRS uncompressed bytes.
+/// This is the hash of the contribution number 15 of the Dusk Trusted Setup.
 pub static CRS_17_HASH: &str =
-    "18b48f588fd4d1e88ef9e7b3cacfa29046f6f489c5c237a4b01ee4f0334772a5";
+    "6161605616b62356cf09fa28252c672ef53b2c8489ad5f81d87af26e105f6059";
 
 const CRS_FNAME: &str = "dev-piecrust.crs";
```

Comment:
Do we want to change the name too? So that our CI doesn't fail due to concurrency

Reply:
done